cmd/vc-rest/startcmd/start.go

```diff
@@ -97,6 +97,10 @@ const (
 		" Alternatively, this can be set with the following environment variable: " + tlsCACertsEnvKey
 	tlsCACertsEnvKey = "VC_REST_TLS_CACERTS"
 
+	universalRegistrarURLFlagName  = "universal-registrar-url"
+	universalRegistrarURLFlagUsage = "Universal Registrar instance is running on. Format: HostName:Port."
```

Comment:
also curious about why we choose to have to in start command instead of an option in profile request?

Reply:
I agree with you it's better to add url as part of profile I will close this PR